packages/web-components/fast-element/docs/guide/declaring-templates.doc.md

```diff
@@ -3,4 +3,191 @@ id: declaring-templates
 title: Declaring Templates
 sidebar_label: Declaring Templates
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-element/docs/guide/declaring-templates.doc.md
----
\ No newline at end of file
+---
+
+## Basic Templates
+
+While you can create and update nodes in the Shadow DOM manually, `FASTElement` provides a streamlined templating system for the most common rendering scenarios. To create an HTML template for an element, import and use the `html` tagged template helper and pass the template to the `@customElement` decorator.
+
+Here's how we would add a template for our `name-tag` component that renders some basic structure as well as our `greeting`:
+
+**Example: Adding a Template to a `FASTElement`**
+
+```ts
+import { FASTElement, customElement, attr, html } from '@microsoft/fast-element';
+
+const template = html<NameTag>`
```

Comment:
Should this template instead leverage semantic HTML rather than DIVs with classes? <header><body><footer>?